at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb

```diff
@@ -0,0 +1,53 @@
+class StringFormat
+  class << self
+  def human_name(string)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/c08836bee3655e104837529ea8ff45ce098ad76f/at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb#L3"></span>Metrics/PerceivedComplexity: Perceived complexity for human_name is too high. [9/8]</td>
    </tr>
  </tbody>
</table>